transactions_test.go

```diff
@@ -224,3 +224,39 @@ func NewTestTransactionFailed() *recurly.Transaction {
 		},
 	}
 }
+
+// Returns a Transaction corresponding to testdata/transaction_refunded.xml
+// as well as the transaction portion of testdata/errors_transaction_refunded.xml.
+func NewTestTransactionRefunded() *recurly.Transaction {
```

Comment:
Do you have an uncommitted test that uses this? Also looks like `testdata/errors_transaction_refunded.xml` is missing from your PR.

This and the corresponding XML file should have `OriginalTransactionUUID` set so we can verify it's extracted out of the XML properly

Reply:
Ah. Thank you for catching that.

Reply:
@cristiangraz addressed feedback.